frontend/ui-pablo/app/components/Organisms/pool/CreatePool/steps/SimilarPoolsStep.tsx

```diff
@@ -78,8 +79,17 @@ const SimilarPoolsStep: React.FC<SimilarPoolsStepProps> = ({
     }
   };
 
-  const token1 = baseAsset !== "none" ? getAsset(baseAsset) : null;
-  const token2 = quoteAsset !== "none" ? getAsset(quoteAsset) : null;
+  const _baseAsset = useMemo(() => {
+    return supportedAssets.find(i => {
+      return i.network[DEFAULT_NETWORK_ID] === baseAsset
+    })
+  }, [baseAsset, supportedAssets])
```

Comment:
The same for baseAsset - It seems that it can be exposed from useStore, instead of having duplicated code.